### ISSUE TITLE
Could not set up andoid phone to work with Maloja

### ISSUE THREAD

**Issue by RCAPDART** [krateng/maloja#28]
Hi!
I've tried to use both two applications: Pano Scrobbler and Simple Scrobbler.
I tried to use http and https. Set up with and without ssl support, with and withouth "http://" and "https://" prefix. Tried several tokens, but still no success.
The only success was with Simple Scrobbler - I set up custom ListenBrainz server with "https://" prefix and disabled ssl.

maloja debug show message:
`API request: ['listenbrainz', '1', 'submit-listens']
Keys:
         listen_type : playing_now

         payload : [{'track_metadata': {'artist_name': 'Benassi, Benny - Best of', 'track_name': 'Satisfaction (Original Album Version)'}}]

         Host : maloja website

         Authorization : token secrettoken

         Content-Type : application/json; charset=UTF-8

         User-Agent : Dalvik/2.1.0 (Linux; U; Android 10; LYA-L29 Build/HUAWEILYA-L29)

         Accept-Encoding : gzip

         X-Forwarded-For : secret

         X-Forwarded-Host : secret

         X-Forwarded-Server : secret

         Connection : Keep-Alive

         Content-Length : 156

         ['submit-listens']

         Response: {'code': 200, 'status': 'ok'}`

But it didn't appear in the Scrobbles page in Maloja.

May be I missed some instruction, of something else?
Thank you in advance!

**Comment by krateng** [comment]
This is just the "playing_now" scrobble, which is accepted but ignored by Maloja. Did you listen to the full song (or at least half of it)? There should be another scrobble of type "single" that causes the actual scrobble to be created.

**Comment by RCAPDART** [comment]
Yes, I left phone for 30 minutes with playing music.

**Comment by RCAPDART** [comment]
It will be great to use Pano. But I don't understand why it is not working.
I set credentials and URL to the custom ListenBrainz account and nothing happens. No success/error messages. When I reopened it - I don't see previously filled data.

Maloja Debug don't show any calls to it.

**Comment by krateng** [comment]
Could you double-check your Pano settings for the custom Listenbrainz server? It should be `https://yourserver.tld/api/s/listenbrainz`

**Comment by RCAPDART** [comment]
![image](https://user-images.githubusercontent.com/1285232/89412154-40decf00-d72f-11ea-94e6-008e53a7c44c.png)
Yes, I set it in this way. May be it is bug, related to the Pano?

**Comment by FoxxMD** [comment]
Just to add a baseline here -- I've got majola working fine on my android phone with Simple Scrobbler and the custom ListenBrainz Server.

![image](https://user-images.githubusercontent.com/4663766/89413906-90090d80-d6f7-11ea-9d3b-fd3df0fd47c9.png)

**Comment by krateng** [comment]
Could you just send a manual request (curl or enter it in your browser) to `https://yourserver.tld/api/s/listenbrainz/1/submit-listens` and see if it show up in the debug log? If it doesn't, I really can only imagine Pano doesn't even try to scrobble for some reason (I use Pano and it works for me).

**Comment by RCAPDART** [comment]
I used "complete submit listen JSON document" from https://listenbrainz.readthedocs.io/en/latest/dev/json/#json-doc. It works fine. Track appeared in the Maloja.

**Comment by krateng** [comment]
So I removed and readded my Maloja in Pano Scrobbler to test it, it works with Maloja 2.6.9 and Pano 1.91. Could you make sure you have those versions and try again?

**Comment by RCAPDART** [comment]
Hi! Finally I made work Simple Scrobbler. But Pano is still don't work. It's ok, I can use Simple Scrobbler. Also - this is showing, that issue is not related to the Maloja. :)
Thank you for your help!

May be sometime it can be helpful for others.

The only one way, when Simple Scrobbler is working on Huawei Mate 20 pro:
1. Put username;
2. Password should not contains "token" word;
3. Both URLS should be the same - "https://{domain-name}.{tld}/api/s/listenbrainz;
4. It should contain "https://" prefix;
5. "Use SSL" should be enabled;

**My server configuration:**
OS: Ubuntu 18;
Python: 3.6.9;
Web-Server: Apache2;
Maloja version: 2.6.9;
Https certificate is provided by LetsEncrypt.

**Android phone configuration:**
Model: Huawei Mate 20 pro (Lya-L29);
Simple Scrobbler version: 1.7.0;
Pano version (not working): latest from the Google Play - 1.91

**What is still not working good**
Native Huawei music player app is showing data in the status bar in format, which is not processed well by the Simple Scrobbler. I received artist name as "{artistName} - {albumName}".
At this case I used foobar2000 application instead. 


Maloja is worked behind the Apache2 (Virtual host with proxy pass).
